plugins/modules/azure_rm_adgroup_info.py

```diff
@@ -55,6 +55,11 @@
             - Indicate whether the groups in which a groups is a member should be returned with the returned groups.
         default: False
         type: bool
+    raw_membership:
+        description:
+            - By default the group_members return property is flattened and partially filtered of non-User objects before return. This argument disables those transformations.
```

Comment:
line too long (175 > 160 characters)
```suggestion
            - By default the group_members return property is flattened and partially filtered of non-User objects before return.\
              This argument disables those transformations.
```